src/features/lightspeed/contentMatchesWebview.ts

```diff
@@ -107,22 +108,82 @@ export class ContentMatchesWebview implements vscode.WebviewViewProvider {
     this._view.webview.html = "";
   }
 
+  public isError(
+    contentMatchResponses: ContentMatchesResponseParams | IError
+  ): contentMatchResponses is IError {
+    return (contentMatchResponses as IError).code !== undefined;
+  }
+
   private async getWebviewContent(): Promise<string> {
-    const noContentMatchesFoundHtml = `<html><body>No training matches found for the latest accepted suggestion.</body></html>`;
+    const noActiveSuggestionHtml = `
+      <html>
+        <body>
+          <p>Training matches cannot be retrieved. No active suggestion found.</p>
```

Comment:
This shows up when you first open the extension. Shall we drop the `Training matches cannot be retrieved.` and say something like `Lightspeed training matches will be displayed here after you accept a Lightspeed inline suggestion.` or `Accept an inline suggestion from Lightspeed to see training matches here.`

Reply:
I've changed to `Training matches will be displayed here after you accept an inline suggestion.`

I avoided mentioning `Lightspeed` to avoid problems with upstream/downstream.